packages/rocketchat-channel-settings/client/views/channelSettings.html

```diff
@@ -94,7 +94,7 @@
 			<div class="rc-user-info__row rc-user-info__row--separator">
 				<div class="rc-switch-double">
 					<div class="rc-switch-double__label  {{equal true value 'disabled'}}">
-						{{_ "Colaborative"}}{{equal default value '*'}}
```

Comment:
@stuartpb  and this was issue number ?

Reply:
Do I really need a separate issue for a *one-character spelling fix*?